main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"flag"
+)
+
+type AssetProxy struct {
+	assetID string
+	proxyID string
+}
+
+func main() {
+	// fmt.Print("Enter your App ID: ")
+	// var app_id string;
+	// fmt.Scanln(&app_id)
+
+	// fmt.Print("Enter your access token: ")
+	// var token string;
+	// fmt.Scanln(&token)
+
+	appIDFlag := flag.String("AppID", "", "Enter your App ID: ");
+	tokenFlag := flag.String("Token", "", "Enter your access token: ");
+	// debugFlag := flag.Bool("DebugFlag", false, "Debugging")
+
+	flag.Parse()
+
+	// fmt.Printf("App ID Value: %s", *appIDFlag)
```

Comment:
Style: remove commented lines 28-29.

Debug statements are removed before pull requests to keep code clean :)